features/step_definitions/basic_steps.rb

```diff
@@ -14,6 +14,10 @@
   click_button button
 end
 
+Given("I click the {string} link") do |link|
+  click_link link
```

Comment:
have you tried `click` instead of `click_link` and `click_button` if that works you could remove this and reuse the step above this one.